saturn-r/Dockerfile

```
@@ -0,0 +1,21 @@
+
+ARG SATURNBASE_IMAGE
+FROM ${SATURNBASE_IMAGE}
+
+COPY environment.yml /tmp/environment.yml
+COPY postBuild /tmp/postBuild.sh
+COPY r-jupyter-kernel /tmp/r-jupyter-kernel.sh 
+# https://stat.ethz.ch/R-manual/R-devel/library/base/html/libPaths.html
+ENV R_LIBS=/usr/local/lib/R/
+
+RUN conda env update -n saturn --file /tmp/environment.yml && \
+    find ${CONDA_DIR} -type f,l -name '*.pyc' -delete && \
+    find ${CONDA_DIR} -type f,l -name '*.a' -delete && \
+    find ${CONDA_DIR} -type f,l -name '*.js.map' -delete
+    
+RUN /bin/bash -e -u -o pipefail /tmp/postBuild.sh && \
+    sudo chmod 755 /tmp/r-jupyter-kernel.sh
```

```suggestion
    sudo chmod 755 /tmp/r-jupyter-kernel.sh && \
    sudo rm /tmp/postBuild.sh
```

It's a tiny thing, but might as well delete the `postBuild` script once you're done with it. To trim a tiny bit off the size of the image.

---

Good call, thank you!